cloud-platform-reports-cronjobs/templates/live-one-domains.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.cronjobs.liveOneDomains.name }}
+spec:
+  schedule: "{{ .Values.cronjobs.liveOneDomains.schedule }}"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 100
+      template:
+        spec:
+          {{- include "cloud-platform-reports-cronjobs.imagePullSecrets" . | indent 10 }}
+          containers:
+          - name: live-one-domains-image
+            image: ministryofjustice/cloud-platform-live-one-domains:{{ .Chart.AppVersion }}
+            imagePullPolicy: Always
+            env:
+            - name: AWS_REGION
+              value: eu-west-2
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: aws-creds
+                  key: access-key-id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: aws-creds
+                  key: secret-access-key
+            {{- include "cloud-platform-reports-cronjobs.kubeconfigLocation" . | indent 12 }}
+            {{- include "cloud-platform-reports-cronjobs.hoodaw-credentials" . | indent 12 }}
+            command:
+            - /bin/sh
+            - -c
+            - ./live-1-domains
```

Comment:
@jackstockley89 , this is incorrect. The module name for the report is `live1-domains` so the binary will be in that name and not `live-1-domains`

Reply:
What was done about this?